CryptoBinPkg/CryptoBinPkg.dsc

```diff
@@ -182,22 +173,34 @@
 #       generated for it, but the binary will not be put into any firmware volume.
 #
 ###################################################################################################
-[Components.IA32, Components.X64] # MU_CHANGE remove ARM and AARCH64
+[Components.IA32, Components.X64]
   CryptoPkg/Driver/CryptoPei.inf {
     <Defines>
-      FILE_GUID = $(PEI_CRYPTO_DRIVER_FILE_GUID)  # MU_CHANGE updated File GUID
+      FILE_GUID = $(PEI_CRYPTO_DRIVER_FILE_GUID)
+  }
+
+  CryptoPkg/Driver/CryptoSmm.inf {
+    <Defines>
+      FILE_GUID = $(SMM_CRYPTO_DRIVER_FILE_GUID)
   }
 
 [Components.IA32, Components.X64, Components.AARCH64]
   CryptoPkg/Driver/CryptoDxe.inf {
     <Defines>
-      FILE_GUID = $(DXE_CRYPTO_DRIVER_FILE_GUID)  # MU_CHANGE updated File GUID
+      FILE_GUID = $(DXE_CRYPTO_DRIVER_FILE_GUID)
   }
 
-[Components.IA32, Components.X64]
```

Comment:
I think we need to also have SMM compiled here.

Reply:
It gets compiled for `[Components.IA32, Components.X64]` up above. Standalone MM is split off as `X64` only because of limitations building `IA32` with some of the MM supervisor libraries.

Please let me know if I misunderstood.

Reply:
I see that now.  The changes shown confused me.